internal/controller/atlasschema_controller.go

```diff
@@ -598,7 +604,83 @@ func (d *managedData) render(w io.Writer) error {
 	if d.Desired == nil {
 		return errors.New("the desired state is not set")
 	}
-	return tmpl.ExecuteTemplate(w, "atlas_schema.tmpl", d)
+	f := hclwrite.NewFile()
+	fBody := f.Body()
+	for _, b := range d.asBlocks() {
+		fBody.AppendBlock(b)
+	}
+	if _, err := f.WriteTo(w); err != nil {
+		return err
+	}
+	return nil
+}
+
+// enableDestructive enables the linting policy for destructive changes.
+// If the force is set to true, it will override the existing value.
+func (d *managedData) enableDestructive(force bool) {
+	check := &dbv1alpha1.CheckConfig{Error: true}
+	destructive := &dbv1alpha1.Lint{Destructive: check}
+	switch {
+	case d.Policy == nil:
+		d.Policy = &dbv1alpha1.Policy{Lint: destructive}
+	case d.Policy.Lint == nil:
+		d.Policy.Lint = destructive
+	case d.Policy.Lint.Destructive == nil, force:
+		d.Policy.Lint.Destructive = check
+	}
+}
+
+// setLintReview sets the lint review policy.
+// If the force is set to true, it will override the existing value.
+func (d *managedData) setLintReview(v dbv1alpha1.LintReview, force bool) {
+	lint := &dbv1alpha1.Lint{Review: v}
+	switch {
+	case d.Policy == nil:
+		d.Policy = &dbv1alpha1.Policy{Lint: lint}
+	case d.Policy.Lint == nil:
+		d.Policy.Lint = lint
+	case d.Policy.Lint.Review == "", force:
+		d.Policy.Lint.Review = v
+	}
+}
+
+// asBlocks returns the HCL block for the environment configuration.
+func (d *managedData) asBlocks() []*hclwrite.Block {
+	var blocks []*hclwrite.Block
+	env := hclwrite.NewBlock("env", []string{d.EnvName})
```

Comment:
The `env` shouldn't have no labels.
```suggestion
	env := hclwrite.NewBlock("env", nil)
```